### DISCUSSION TITLE
Usability Bug: Add instant delete/ option for the cards on Team's Dashboard & Archive

### DISCUSSION THREAD

**Discussion by elielytz** [ParabolInc/parabol#5959] [New features]
## Current Feature state

- Dragging Cards along Dashboard
- Edit Cards
- Move from one stack to another
- Archive
- Return Card From Archive

<img src="https://user-images.githubusercontent.com/45159239/149948449-ed7a29ba-c7d1-4bf2-ab7b-19ab32f4a9e7.png" width=35% height=35%>

## Improvement Suggestions

- Add Instant delete card option 

## Advantage 

- This will remove unnecessary cards in the team's dashboard & in an archive.

**Comment by jordanh** [comment]
Thank you opening this. Fun fact! Although clunky, you can select all the card content, delete it, then blur the card (by clicking elsewhere) and the card will delete

**Comment by elielytz** [comment]
Thanks @jordanh it works like charm and disappears when I delete all contents, 
So in later updates is there any consideration to add information that directs user to **_"select all the card content, delete it, then blur the card "_** so that can make the card disappear?

**Comment by ackernaut** [comment]
Thanks for adding this! We marked this as an enhancement instead of a bug since it’s an improvement to existing, designed behavior, not a degraded experience of designed behavior
